terminus/geometry/latlon.py

```diff
@@ -15,6 +15,7 @@
 """
 
 import math
+import numpy as np
 
```

Comment:
We should definitely separate the classes into their own files. See http://stackoverflow.com/questions/744373/circular-or-cyclic-imports-in-python on how to handle cicrular imports.